From d843a5aaa5d6bb2e9868a32ecb1af0278f8db0d2 Mon Sep 17 00:00:00 2001
From: Zhao Wei Liew <zhaoweiliew@gmail.com>
Date: Sat, 21 Jan 2017 17:50:06 +0800
Subject: [PATCH 03/18] Settings: gesture: Add LineageParts touchscreen gesture
 settings

We'll allow devices to add touchscreen gesture settings
using the LineageHW TouchscreenGesture API.

Change-Id: I569b91ed66d34097a2bd96fb56af660a2702224b
---
 res/xml/gestures.xml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/res/xml/gestures.xml b/res/xml/gestures.xml
index 650b1c4400..ef5b733e51 100644
--- a/res/xml/gestures.xml
+++ b/res/xml/gestures.xml
@@ -17,10 +17,15 @@
 
 <PreferenceScreen
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:lineage="http://schemas.android.com/apk/res/lineageos.platform"
     xmlns:settings="http://schemas.android.com/apk/res-auto"
     android:key="gesture_settings_screen"
     android:title="@string/gesture_preference_title">
 
+    <org.lineageos.internal.lineageparts.LineagePartsPreference
+        android:key="touchscreen_gesture_settings"
+        lineage:requiresFeature="lineagehardware:FEATURE_TOUCHSCREEN_GESTURES" />
+
     <Preference
         android:key="gesture_assist_input_summary"
         android:title="@string/assist_gesture_title"
-- 
2.17.1

